### PR TITLE
[dpes] Upgraded dependency

### LINKED ISSUE #103: [dependencies] Update cryptography to 3.1.1
Update the cryptography library to allow the latest version, ensure tests pass and the main features work correctly.

### PR DESCRIPTION
**Pull request by DaffyTheDuck**
Updated cryptography to latest(3.1.1)

closes #103

### CODE REVIEW

**Comment by DaffyTheDuck** [comment]
Tested the CA and CERT with new dep. worked fine :smile: `tests` passed without any error on local machine :smile:

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 django>=2.2,<3.2
 django-model-utils>=4.0
 jsonfield>=3.1.0,<4.0.0
-cryptography>=2.4.0,<3.1.0
+cryptography>=2.5.0,<3.2.0
```
Comment (NoumbissiValere) [inline requirements.txt:4]: @DaffyTheDuck since we are updating to version 3.1.1, why don't you write this as 
`cryptography>=3.1.1,<3.2.0`

**Review by NoumbissiValere — CHANGES_REQUESTED** [review]
Good progress @DaffyTheDuck

File: requirements.txt (hunk 1 above)
Comment (DaffyTheDuck) [reply requirements.txt:4]: Sure!

**Review by NoumbissiValere — APPROVED** [review]
Great job @DaffyTheDuck . This looks good to me. Leaving for @nemesisdesign to verify and if possible merge with the appropriate label 👍

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**